thorlcr/mfilemanager/thmfilemanager.hpp

```diff
@@ -40,6 +40,10 @@
 
 #define PAUSETMPSCOPE "thorpause"
 
+#define DEFAULTXMLROWTAG "Row"
+#define DEFAULTXMLHEADER "<Dataset>"
+#define DEFAULTXMLFOOTER "</Dataset>"
```

Comment:
As mentioned in other comment, these would be better place in common file and used by hthor/roxie too.